packages/host/app/components/operator-mode/overlays.gts

```diff
@@ -143,22 +143,24 @@ export default class OperatorModeOverlays extends Component<Signature> {
       .overlay-embedded-card-header {
         background: var(--boxel-light-100);
         height: var(--overlay-embedded-card-header-height);
+        display: flex;
+        padding: 13px;
```

Comment:
@jurgenwerk do we still need this padding? if so can we express in terms of var(--boxel-sp)?

Reply:
Addressed in https://github.com/cardstack/boxel/pull/491/commits/9718a85783d0cd28cda2bb618204dafefc5b1d35